Tests/RevenueCatUITests/Helpers/TestCase.swift

```diff
@@ -32,20 +32,6 @@ class TestCase: XCTestCase {
         XCTestObservationCenter.shared.removeTestObserver(CurrentTestCaseTracker.shared)
     }
 
-    // swiftlint:disable unneeded_override
-
-    @MainActor
-    override func setUpWithError() throws {
-        try super.setUpWithError()
-    }
-
-    @MainActor
-    override func tearDown() {
-        super.tearDown()
-    }
-
-    // swiftlint:enable unneeded_override
-
```

Comment:
In main, we only had to remove the comments to make the linter happy, but locally SwiftLint was directly removing the two functions when I removed the comments.

https://github.com/RevenueCat/purchases-ios/blob/main/Tests/RevenueCatUITests/Helpers/TestCase.swift#L35-L43